src/pages/Overview/OverviewPage.tsx

```diff
@@ -38,7 +37,7 @@ interface OverviewProps {
   setActiveNamespace(name: Namespace): (namespace: Namespace) => void;
 }
 
-class OverviewPage extends ListPage.Component<OverviewProps, State, NamespaceInfo> {
+class OverviewPage extends React.Component<OverviewProps, State> {
   private static summarizeHealthFilters() {
```

Comment:
I'm a bit confused by `summarizeHealthFilters()` function being static, but it's calling  `FilterSelected.getSelected()` which is part of StatefulFilters.tsx.  Does `FilterSelected.getSelected()` maintain state?

Reply:
yes, this is a global static state. It could be a candidate for redux (I think it was introduced before we started to use redux), but at the moment it's not, and to be honest it works quite well